lib/hammer_cli_foreman/host.rb

```diff
@@ -382,9 +382,6 @@ def validate_options
             validator.all(:option_environment_id, :option_architecture_id, :option_domain_id,
                           :option_puppet_proxy_id, :option_operatingsystem_id,
```

Comment:
as well as `option_environment_id`, this would help being able to provision without puppet things (although the workaround is using a hostgroup)